cypress/component/Tabs.spec.tsx

```diff
@@ -75,7 +75,7 @@ describe('Tabs', () => {
 
         context('when the tab key is pressed', () => {
           beforeEach(() => {
-            cy.tab();
+            cy.realPress('Tab');
```

Comment:
realPress or realType?

Reply:
There's no `realType` for the tab key. There's only a `realPress`. It probably makes more sense to use `realPress` for keyboard commands and `readType` or just `type` for entering in text.